features/openhab-addons-external/src/main/resources/conf/gcal.cfg

```diff
@@ -0,0 +1,10 @@
+# Credentials for Google API and Google Calendar name
+
+# Copied from Credentials page on console.developers.google.com
+#client_id=
+
+# Copied from Credentials page on console.developers.google.com
+#client_secret=
+
+# The name of the Google Calendar that GCal should use
+#calendar_name=
```

Comment:
> openhab_default.cfg contained the old (pre-OAuth) config parameters

I was hoping this PR would be a good opportunity for consistency between the now two .cfg files, seeing as you report that we missed updating openhab_default.cfg file with @SirAd's big refresh.